examples/shap/binary_classification.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+import numpy as np
+from sklearn.datasets import load_breast_cancer
+from sklearn.ensemble import RandomForestClassifier
+
+import mlflow
+from utils import to_pandas_Xy, show_image
+
+
+# prepare training data
+X, y = to_pandas_Xy(load_breast_cancer())
+X = X.iloc[:50, :8]
+y = y.iloc[:50]
+
+# train a model
+model = RandomForestClassifier()
+model.fit(X, y)
+
+# log an explanation
+with mlflow.start_run() as run:
+    mlflow.shap.log_explanation(lambda X: model.predict_proba(X)[:, 1], X)
+
+# list artifacts
+client = mlflow.tracking.MlflowClient()
+artifact_path = "model_explanations_shap"
+artifacts = [x.path for x in client.list_artifacts(run.info.run_id, artifact_path)]
+print("# artifacts:")
+print(artifacts)
+
+# load back the logged explanation
+dst_path = client.download_artifacts(run.info.run_id, artifact_path)
+base_values = np.load(os.path.join(dst_path, "base_values.npy"))
+shap_values = np.load(os.path.join(dst_path, "shap_values.npy"))
+
+print("\n# base_values")
+print(base_values)
+print("\n# shap_values")
+print(shap_values[:3])
```

Comment:
Could we also show usage of the loaded base values & shap values? e.g. to generate explanations on new input to the model?

Reply:
We could generate a new plot using the loaded base values and shap values like:

```
shap.force_plot(base_values, shap_values[0, :], X.iloc[0, :])
```